server/src/main/java/com/google/musicanalysis/cache/AnalysisCache.java

```diff
@@ -32,6 +34,8 @@
 /** Implementation of server side cache that stores API requests to save time and API quota. */
 public class AnalysisCache {
   private static final String CACHE_FILE = "cachedData.txt";
+  private static final long ONE_DAY_IN_SECONDS = 86400;
```

Comment:
nit: `ONE_DAY_IN_SECONDS = 24 * 60 * 60`

Reply:
Please make this change.